cli/config/plugin.webpack.config.js

```diff
@@ -275,33 +272,25 @@ module.exports = ({ env: webpackEnv, config, paths }) => {
                                 },
                             }),
                         },
-                        // dhis2: 'asset/resource' fixes fonts, but 'file-loader' breaks css modules
-                        // when used for all asset types. So use each for respective files
-                        {
-                            test: /\.(woff|woff2|eot|ttf|otf)$/i,
-                            type: 'asset/resource',
-                            generator: {
-                                filename: 'static/media/[name].[hash][ext]',
-                            },
-                        },
                         // "file" loader makes sure those assets get served by WebpackDevServer.
                         // When you `import` an asset, you get its (virtual) filename.
                         // In production, they would get copied to the `build` folder.
                         // This loader doesn't use a "test" so it will catch all modules
                         // that fall through the other loaders.
                         {
-                            loader: require.resolve('file-loader'),
                             // Exclude `js` files to keep "css" loader working as it injects
                             // its runtime that would otherwise be processed through "file" loader.
                             // Also exclude `html` and `json` extensions so they get processed
-                            // by webpacks internal loaders.
+                            // by webpack's internal loaders.
                             exclude: [
+                                /^$/,
```

Comment:
I believe this newly added regex only matches an empty string. I wonder if this is actually useful, or perhaps a remnant of some early stage experimentation.....
<img width="172" alt="Screenshot 2023-02-16 at 15 05 24" src="https://user-images.githubusercontent.com/353236/219386914-a704d928-c373-4364-9370-e1b6e9c6c157.png">



Could you explain why this is needed here?